### PR TITLE
DEVX-1321: Schema Validation fails for Replicated topic

### PR DESCRIPTION
**Pull request by ybyzek**
Ensure schema is created before Replicator starts.  Otherwise schema validation fails and the Replicator task fails

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -733,25 +733,27 @@ solution, Confluent Replicator is also configured with security.
 
    .. figure:: images/replicator_topic_info.png
 
+5. Notice that because |crep| default is ``topic.config.sync=true``, then the replicated
```
Comment (rspurgeon) [inline docs/index.rst]: I had to go look up the `topic.config.sync` docs to be certain I undetstood what it does.  The name is decently self-explanatory but I wanted to be sure..  For you consideration, a link or quick text of what this setting accomplishes might help a future reader.

Comment (ybyzek) [reply docs/index.rst]: Good suggestion, done

File: docs/index.rst [hunk 2]
```diff
@@ -733,25 +733,27 @@ solution, Confluent Replicator is also configured with security.
 
    .. figure:: images/replicator_topic_info.png
 
+5. Notice that because |crep| default is ``topic.config.sync=true`` (see |crep| :ref:`documentation <rep-destination-topics>`), then the replicated
+   topic ``wikipedia.parsed.replica`` has enabled |sv| just like the original
+   topic ``wikipedia.parsed``.
```
Comment (rspurgeon) [inline docs/index.rst]:
```suggestion
   topic ``wikipedia.parsed`` (you may need to click on the ``Show full config`` button to see all values).
```

Comment (ybyzek) [reply docs/index.rst]: Great suggestion @rspurgeon .  I modified it slightly but added the gist.

**Pull request merged**